modules/persistent-storage-local-install.adoc

```diff
@@ -49,3 +49,82 @@ $ oc annotate project local-storage openshift.io/node-selector=''
 .. Click *Subscribe*.
 
 . Once finished, the Local Storage Operator will be listed in the *Installed Operators* section of the web console.
+
+.. Navigate to *Operators* -> *OperatorHub*.
+
+.. Type *Local Storage* into the filter box to locate the Local Storage Operator.
+
+.. Click *Install*.
+
+.. On the *Create Operator Subscription* page, select *A specific namespace on the cluster*. Select *local-storage* from the drop-down menu.
+
+.. Adjust the values for the *Update Channel* and *Approval Strategy* to the desired values.
+
+.. Click *Subscribe*.
+
+. Once finished, the Local Storage Operator will be listed in the *Installed Operators* section of the web console.
+
+.From the CLI
+. Install the Local Storage Operator from the CLI.
+
+.. Create an object YAML file to define a Namespace, OperatorGroup, and Subscription for the Local Storage Operator,
+such as `local-storage.yaml`:
++
+.Example local-storage
+[source,yaml]
+----
+  apiVersion: v1
+  kind: Namespace
+  metadata:
+    name: local-storage
+  ---
+  apiVersion: operators.coreos.com/v1alpha2
+  kind: OperatorGroup
+  metadata:
+    name: local-operator-group
+    namespace: local-storage
+  spec:
+    targetNamespaces:
+      - local-storage
+  ---
+  apiVersion: operators.coreos.com/v1alpha1
+  kind: Subscription
+  metadata:
+    name: local-storage-operator
+    namespace: local-storage
+  spec:
+    channel: "stable-{product-version}" <1>
```

Comment:
@qinpingli I found this in our docs: https://docs.openshift.com/container-platform/4.2/updating/updating-cluster-between-minor.html#understanding-upgrade-channels_updating-cluster-between-minor

So I added `stable-` in channel version. OK?

Reply:
@bobfuru These channels should be for payload image, for 3rd level operators, the channels are `4.2`, '4.3', ...

Reply:
Ah, I see @qinpingli - thanks for clarifying. Changed it back to just `{product-version}` (dropped `stable-`). Does everything look good to you now?